hcn/hcnglobals.go

```diff
@@ -61,6 +61,8 @@ var (
 	}
 	// HNS 13.0 allows for Set Policy support
 	SetPolicyVersion = VersionRanges{VersionRange{MinVersion: Version{Major: 13, Minor: 0}, MaxVersion: Version{Major: math.MaxInt32, Minor: math.MaxInt32}}}
+	// HNS 15.0 allows for Modify Policy support
+	ModifyPolicyVersion = VersionRanges{VersionRange{MinVersion: Version{Major: 15, Minor: 0}, MaxVersion: Version{Major: math.MaxInt32, Minor: math.MaxInt32}}}
```

Comment:
Major version 15 is pretty old ~2021 timeframe. You may need to bump up the minor version once the related changes are checked in HNS.

Reply:
Can we just set this to a next version value a-priori before HNS changes are checked in?

Reply:
This is the hns version I received in KubeProxy from latest AKS

**Node details:**
K8S Version:  v1.28.9 
Windows Server 2022 Datacenter   10.0.20348.2402    containerd://1.7.14+azure

**hns version**="{Major:13 Minor:3}"